.nvmrc

```diff
@@ -1 +1 @@
-4.8.4
\ No newline at end of file
+6.0.0
```

Comment:
I think you should use the last version of node 6 as like the one used in Circleci

Reply:
I think we should use the lowest possible instead. I updated the circleci config to use node v6.11.1 (6.0.0 is not available...)